src/index.html

```diff
@@ -13,6 +13,8 @@
     <noscript
       >Javascript appears to be disabled - unfortunately the estimator requires it to be enabled to function.</noscript
     >
-    <tech-carbon-estimator></tech-carbon-estimator>
+    <div class="tce-px-4 tce-pt-4 pd:tce-pb-4">
```

Comment:
```suggestion
    <div class="tce-pt-4 md:tce-pb-4">
```
Assume typo on the bottom padding and I'm thinking we actually don't want the x padding here as the SL site is using this version within the iframe. We'll just have to be aware that it will look very close to the sides when testing in isolation.

Reply:
It hadnt quite clicked that the SL site was using this as the iframe. I dont wonder if we need any padding then?